package.json

```diff
@@ -159,12 +161,14 @@
     "analyze-bundles": "webpack-bundle-analyzer stats.json public -p 9898",
     "autoprefixer": "postcss -r --use autoprefixer --autoprefixer.browsers \"last 2 versions, > 1%, Safari >= 8, iOS >= 8, Firefox ESR, Opera 12.1\"",
     "prebuild": "npm run -s install-if-deps-outdated",
-    "build": "run-p -s build-devdocs:* build-server build-css",
+    "build": "run-p -s build-devdocs:* build-server build-loadcss build-css",
     "postbuild": "npm run -s bundle",
+    "build-loadcss": "uglifyjs node_modules/fg-loadcss/src/loadCSS.js node_modules/fg-loadcss/src/cssrelpreload.js -m -c -o public/loadcss.m.js",
     "build-css": "run-p -s build-css:*",
     "build-css:debug": "node-sass --include-path client --source-map \"public/style-debug.css.map\" assets/stylesheets/style.scss public/style-debug.css && npm run -s autoprefixer -- public/style-debug.css",
     "build-css:directly": "node-sass --include-path client assets/stylesheets/directly.scss public/directly.css && npm run -s autoprefixer -- public/directly.css",
     "build-css:editor": "node-sass --include-path client assets/stylesheets/editor.scss public/editor.css && npm run -s autoprefixer -- public/editor.css",
+    "build-css:shell": "node bin/generate-shell-css.js",
```

Comment:
There's no guarantee that `style.css` is generted at this point. You should put it in `postbuild-css`.